src/components/NewsletterForm/index.tsx

```diff
@@ -78,8 +78,7 @@ export const NewsletterForm = ({
                             trust me you don’t want to miss a thing.
                         </p>
                         <p className="opacity-80 mt-2 text-sm">
-                            Plus if you decide that these emails aren’t brightening your day, you can unsubscribe at any
-                            time, no hard feelings.
+                            Don't worry, you can unsubscribe at any time, no hard feelings.
```

Comment:
This copy doesn't appear visible on the Netlify preview, so there might be an issue here.

Reply:
No, this is correct.